server/processors-zipkin/src/test/java/org/hawkular/apm/server/processor/zipkin/CompletionTimeProcessingDeriverTest.java

```diff
@@ -75,15 +75,15 @@ public void testSpanTraceCompletionTime() throws RetryAttemptException {
         descendant.setId("descendant");
         descendant.setTraceId("trace");
         descendant.setTimestamp(1500L);
-        Mockito.when(spanCacheMock.getTrace(null, "trace")).thenReturn(
-                new HashSet<>(Arrays.asList(rootSpan, descendant)));
+        Mockito.when(spanCacheMock.getTrace(null, "trace"))
+                .thenReturn(new HashSet<>(Arrays.asList(rootSpan, descendant)));
 
         completionTimeProcessing = new CompletionTimeProcessing(rootSpan);
         completionTimeProcessing = completionTimeProcessingDeriver.processOneToOne(null, completionTimeProcessing);
         Assert.assertNull(completionTimeProcessing.getCompletionTime());
 
         completionTimeProcessing = completionTimeProcessingDeriver.processOneToOne(null, completionTimeProcessing);
-        Assert.assertEquals(2, completionTimeProcessing.getCompletionTime().getDuration());
+        Assert.assertEquals(2000, completionTimeProcessing.getCompletionTime().getDuration());
```

Comment:
In this case it is better to change completion time, as the Span units were not changed.